codice/ddf
# DDF-2027: Small Logviewer Improvements

#### What does this PR do?
Small improvements to the LogViewer in the Admin UI
  - Reordered log level filters to match karaf levels
  - Added a link to open the viewer in a new tab
  - Renamed export button to be less ambiguous

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@andrewkfiedler @djblue @tbatie 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@figliold
@lessarderic

#### How should this be tested?
Build, run, verify that:
 - Export button under System says "Export System Configuration"
 - The new tab link on the LogViewer shows when in an iframe, but not fullscreen
 - The Log Viewer's log level drop-down menu is correctly ordered

#### Any background context you want to provide?
  We did not move the export button under the configuration tab because the code used to generate that tab is reused in many other pages in the ui. Allowing for this specific tab to display an export button would result in a large amount of code rewriting for very little value. Since the problem we were trying to solve was the button's ambiguity, we decided that the best thing to do was to relabel it to say "Export System Configurations" to solve that issue.

#### What are the relevant tickets?
DDF-2027

#### Screenshots
![image](https://cloud.githubusercontent.com/assets/12499654/14501048/606c96f0-0159-11e6-9b9c-ed0734089600.png)
